dds/DCPS/MultiTopicDataReaderBase.cpp

```diff
@@ -274,29 +264,23 @@ void MultiTopicDataReaderBase::data_available(DDS::DataReader_ptr reader)
       if (gen.info_[i].valid_data) {
         incoming_sample(gen.samples_[i], gen.info_[i], topic.c_str(), meta);
       } else if (gen.info_[i].instance_state != ALIVE_INSTANCE_STATE) {
-        DataReaderImpl* resulting_impl =
-          dynamic_cast<DataReaderImpl*>(resulting_reader_.in());
-
+        DataReaderImpl* resulting_impl = dynamic_cast<DataReaderImpl*>(resulting_reader_.in());
```

Comment:
I'd like to avoid having these kinds of changes in the revision history.  It's not actually changing anything, and it takes some effort on the part of the reader to figure that out.

Reply:
Alright, I will try to keep that in mind.